portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,15 +28,33 @@
 public class DataServlet extends HttpServlet {
 
   private List<String> messages = new ArrayList<String>();
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+    //Get input from form.
+    String text = getParameter(request, "comments", "");
+
+    //Populate List with the data
+    messages.add(text);
+    
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println(messages);
```

Comment:
Should we send back the messages as JSON serialized?

Reply:
This is what I was a bit confused about, since JSON is used in a get request, would I use the same functionality to POST as well?